### ISSUE TITLE
NTR, pulmo: Nasal dryness

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6477]
**Preferred term label:**
Nasal dryness

**Synonyms**

**Definition (free text, please give PubMed ID)**
A lack of humidification of the nasal mucosa.
Comment: A dry nose can be associated with multiple causes, for example an underlying disease, medication or age-related changes in nasal physiology and structure.
•	PMID: 20465792
•	PMID: 12391797

**Parent term (use hpo.jax.org/app)**
Functional respiratory abnormality HP:0002795
Abnormality of the nasal mucosa HP:0000433

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
lmu: @katrinknoflach, @mgriese

**Comment by pnrobinson** [comment]
I would classify this as morphological -- while itt has functional consequences, it seems the observable phenotype is morphological?

**Comment by pnrobinson** [comment]
added
